test/parallel/test-event-emitter-remove-all-listeners.js

```diff
@@ -38,24 +38,25 @@ function expect(expected) {
 
 {
   const ee = new events.EventEmitter();
-  ee.on('foo', common.noop);
-  ee.on('bar', common.noop);
-  ee.on('baz', common.noop);
-  ee.on('baz', common.noop);
+  const noop = common.mustNotCall();
```

Comment:
Why not bring this up to the parent scope and use it in the other places below that are now manually calling `common.mustNotCall()`?